scripts/skills/perks/perk_rf_unstoppable.nut

```diff
@@ -80,14 +80,8 @@ this.perk_rf_unstoppable <- ::inherit("scripts/skills/skill", {
 			return;
 		}
 
-		if (this.m.Stacks >= this.m.MaxStacks)
-			return;
-
 		local actor = this.getContainer().getActor();
-		if (actor.getActionPoints() <= actor.getActionPointsMax() / 2)
-		{
-			this.m.Stacks++;
-		}
+		this.m.Stacks = actor.getActionPoints() <= actor.getActionPointsMax() / 2 ? ::Math.min(this.m.MaxStacks, this.m.Stacks + 1) : 0;
```

Comment:
I would put `(actor.getActionPoints() <= actor.getActionPointsMax() / 2)` like this in brackets. It's a mouthful of a condition that makes this one-liner harder to parse.
This comment is mainly to archive this line of code as an example in case we discuss the use of optional brackets.

Reply:
I've changed it to an `if/else` to improve readability in this case.